### PR TITLE
Single Lesson: Polish the design

### LINKED ISSUE #2638: Single Lesson: Polish the design
There’s some design work for the full-screen lessons that lives [in Figma here](https://href.li/?https://www.figma.com/design/ZKaf6u8mIHkAanluWafXAp/Learn?node-id=3473-34133&t=19mV6GnRSxbQzM21-1), that could be worth looking at. I realize it may be limited what you can do here, but there are some things about icons, positioning, sizes, etc, that could benefit from small tweaks.

Image

### LINKED ISSUE #2809: Clearer lesson completed indicator on the single lesson page
The lesson completed indicator was removed in https://github.com/WordPress/Learn/pull/2806 due to the a11y issue.
Currently it needs a further design based on the discussion starting from [this comment](https://github.com/WordPress/Learn/pull/2806#issuecomment-2263389133).

### PR DESCRIPTION
**Pull request by outdoor2kode**
Resolves #2638 #2809

This PR goes through the breadcrumbs, sidebar nav, and the main content style to make them align with the latest design, for both non-standalone and standalone lessons.

**Non-Standalone**
| Nav | Main Content |
|-|-|
| Screenshot 2024-08-26 at 13 26 13 | Screenshot 2024-08-26 at 14 17 45 |

| Rounded Progress Bar |
|-|
| image |

**Standalone**
image

### CODE REVIEW

**Comment by jasmussen** [comment]
Thanks for keeping on this. Happy to defer to Francisco on the color styles, where the green doesn't match the blue from the referenced issue, but was introduced by the larger visual refresh. Can work well.

I know this is probably tricky, but would it be possible to get the rounded pill-shape on the green bar here?

Screenshot 2024-08-26 at 14 30 17

**Comment by outdoor2kode** [comment]
image

Does something like this fit? Only the top-right and bottom-right are made rounded.

**Comment by jasmussen** [comment]
Yes, exactly. Nice.

**Comment by adamwoodnz** [comment]
Alignment of the dropdown arrows and completion icons still looks off to me. The design shows them aligned with the first line

| Design | This PR |
|--------|--------|
| Screenshot 2024-08-27 at 12 20 11 PM | Screenshot 2024-08-27 at 12 14 02 PM |

**Review by adamwoodnz — COMMENTED** [review]
Nice work cleaning this up! One comment on the sidebar icon alignment above.

**Comment by adamwoodnz** [comment]
> image
> Does something like this fit? Only the top-right and bottom-right are made rounded.

I noticed the design has the progress bar wider than the border too

Screenshot 2024-08-27 at 12 23 21 PM

**Comment by outdoor2kode** [comment]
> I noticed the design has the progress bar wider than the border too

I feel that it can be as wide as the border just like other progress bars are. What do you think? @fcoveram

**Comment by jasmussen** [comment]
> I noticed the design has the progress bar wider than the border too

I think Francisco changed this, and I'm happy to defer to him. No blockers on that detail if this was part of the design.

**Comment by fcoveram** [comment]
I have pending the update of those mockups. There are a few changes that worth addressing all together (including #2638 and #2809)

**Comment by outdoor2kode** [comment]
> Alignment of the dropdown arrows and completion icons still looks off to me. The design shows them aligned with the first line

4797aea & df3c906

**Comment by outdoor2kode** [comment]
**Header** and **Sidebar** Updates based on the latest [design](https://www.figma.com/design/ZKaf6u8mIHkAanluWafXAp/Learn?node-id=1705-79&t=QvAxcnBQBSl4edg0-0) 
878d3c1, 37a4bc3, 38d0f6c


|| Module | Standalone |
|-|-|-|
| Desktop | Screenshot 2024-09-02 at 01 17 02 | Screenshot 2024-09-02 at 01 17 10 |
| Tablet | image | image |
| Mobile | image | image |

**Comment by outdoor2kode** [comment]
**Actions** Style Updates
a8200dd, 00e0e1a, 8c38d97

|| Next Lesson | Take Quiz | Complete |
|-|-|-|-|
| Desktop | Screenshot 2024-09-02 at 03 18 42 | Screenshot 2024-09-02 at 03 15 52 | Screenshot 2024-09-02 at 03 16 14 |
| Mobile | Screenshot 2024-09-02 at 03 19 30 | Screenshot 2024-09-02 at 03 19 16 | Screenshot 2024-09-02 at 03 19 40 |

~The style and design for the "Complete Lesson" are not yet consistent. I have left a comment in Figma and will wait for feedback before making further adjustments.~

For the [quiz single page actions](https://www.figma.com/design/ZKaf6u8mIHkAanluWafXAp/Learn?node-id=5492-14630&t=QvAxcnBQBSl4edg0-0), I will open another PR to address their style updates, as in Sensei, the components there cannot easily be separated and moved into Learn without making things overly complex. Creating a PR directly in Sensei will be cleaner and easier to understand. ([Issue](https://github.com/WordPress/Learn/issues/2871) was created for tracking)

**Update**
1. Always display "Next/Previous lesson" buttons as per [discussion](https://www.figma.com/design/ZKaf6u8mIHkAanluWafXAp?node-id=5492-14466#932604124) aecb969 4d62a3d

2. Green background color 4f8b11c
Screen Capture on 2024-09-02 at 23-47-18

**Comment by outdoor2kode** [comment]
**Suggestions Block Reposition**
73ce7b1, a43274f, f3d52ae

| Desktop | Tablet | Mobile |
|-|-|-|
| image | image | image |

**Comment by outdoor2kode** [comment]
**Login Block Reposition** (*styles of action buttons on either desktop or mobile view will be fixed in the following commits)
8aca602, 2ed7dcb

| Desktop | Tablet | Mobile |
|-|-|-|
| Screenshot 2024-09-02 at 04 08 57 | image | Screenshot 2024-09-02 at 04 09 46 |

**Comment by outdoor2kode** [comment]
**Completed Notice**
89e409e

| Desktop | Tablet | Mobile |
|-|-|-|
| Screenshot 2024-09-03 at 05 09 09 | Screenshot 2024-09-03 at 05 09 15 | Screenshot 2024-09-03 at 05 09 25 |

**Comment by outdoor2kode** [comment]
**Sign-in Block Updates**

This will be addressed in another PR, as it's slightly more complicated.

**Comment by outdoor2kode** [comment]
**Preview Mode**

Pending [feedback](https://www.figma.com/design/ZKaf6u8mIHkAanluWafXAp?node-id=5492-14290#933806892).

**Comment by outdoor2kode** [comment]
@adamwoodnz I think this PR is ready for review again, and you can start from [this comment](https://github.com/WordPress/Learn/pull/2862#issuecomment-2323358118). I have documented all the changes that are based on [the latest design](https://www.figma.com/design/ZKaf6u8mIHkAanluWafXAp/Learn?node-id=1705-79&node-type=CANVAS&t=EM97Ys8uJ2Udpjhp-0) in separate comments, each with screenshots and relevant commits. Thanks 🙏 

The following items will be handled separately later:

* **Single quiz updates** will be handled in a separate PR.
* **Sign-in block styles** will be handled in a separate PR.
* **Facilitator Notes** will be handled in a separate PR.
* **Preview mode indicator** is pending feedback.

**Comment by adamwoodnz** [comment]
The design shows the h1 in Inter now

| Design | This branch |
|--------|--------|
| Screenshot 2024-09-03 at 10 47 47 AM | Screenshot 2024-09-03 at 10 47 18 AM |

**Comment by adamwoodnz** [comment]
Little shift in the progress bar width on iPad when activating the menu:

https://github.com/user-attachments/assets/5cac84da-4fe4-4b96-b25d-c97b183373b9

And the buttons don't take up the full width of the wrapper, so when they are active/focused they look like this:

Screenshot 2024-09-03 at 10 55 46 AM

**Review by adamwoodnz — COMMENTED** [review]
This is looking great! A few minor things I found are noted inline 🚀

**Comment by outdoor2kode** [comment]
> Little shift in the progress bar width on iPad when activating the menu:

I thought this was the original design—let me fix it.

**Comment by outdoor2kode** [comment]
> The design shows the h1 in Inter now

0dc0f03
image

> Little shift in the progress bar width on iPad when activating the menu:

06eeda5
image

> And the buttons don't take up the full width of the wrapper, so when they are active/focused they look like this:

3bdadac
image

**Comment by outdoor2kode** [comment]
This PR has quite a few commits. If there are no issues at the moment, I plan to merge it first and handle the remaining parts and the latest design updates in other PRs.

**Review by adamwoodnz — APPROVED** [review]
LGTM! Great work 🎸

**Pull request merged**
